### ISSUE TITLE
Marketing task

### ISSUE THREAD

**Issue by lukewegryn** [ResistancePlatform/grants#3]
This is a proposal for a Resistance marketing task:

**A basic marketing activity for 8-weeks, focused on increasing the number of miners and the ResDEX user-base. Where a roadmap, estimated daily exercise, and the marketing channels (e.g. Reddit, Discord, etc.) should be presented to us.**

If this proposal is approved by the Masternode voters, then the community member who performs this task will be issued a grant from the developer's fund (from the 10% reward splitting).

The amount of RES granted will be: **89,600 RES**

A simple majority will be used to approve or reject this proposal. At least one Masternode must vote on an issue for it to be approved. To apply with a proposal email admin [at] resistance.io and include your GitHub username in your email so our admins can verify your development record. **For information on how to cast your vote, please refer to the README in this repository.**

Voting opens immediately and will close on: **Friday, June 26, 2020 at 23:59 UTC**

**Comment by lukewegryn** [comment]
-----BEGIN VOTE-----
Y:3:r16bGySCm5ycJFbqaH1eN4pKuEcaecXBRzA:1592926614:H75MVXvyCEPko6WT48WA0fRAVozKQbS7S2Y4tPhGJ7xBZxyCmfC03rFa/cEFcnFyvAC4Aj9CM8kBaiAdmdvwv2g=
-----END VOTE-----

**Comment by whiteknightrader** [comment]
-----BEGIN VOTE-----
Y:3:r17bxNVqW8o2BmXy6CEZnkDDyXcTHFWyQbB:1592932858:H/07cxON2G8vRo0oPKz9i5pJpmmqjyieaxdKm4c59u0ldoJQhNsgOv2/1VtATtINVkofAyc6UOEvch9jr32w67Q=
-----END VOTE-----

**Comment by MEtoolel** [comment]
Y:3:r18fiaRwkHZNy2n37wRLrH4SuFTfJJCS2HV:1592983356:H0t8hSMbwLPCpHA2K+x6bkGen/BCgBkonsODj9qyiC4KP6mPAiV+Gd7o/A/piOjP4k0yUAIybmoV3Z+FlDMZ+Lo=

**Comment by ghost** [comment]
-----BEGIN VOTE-----
Y:3:r13fwb1K9sK6YYoTf22c6CJPxequKMxcA7L:1592983765:HzA1YJdWX00EUpFm19+0DasO6GRDtfBPiYMLWVoOZd6pGO+w2k8fN/oRHfb1CXCm0rwpXCWeq1wJNgZyQfvXYA4=
-----END VOTE-----

**Comment by lukewegryn** [comment]
The votes have been tallied and the results are in favor of accepting this proposal.

```
cat list_of_votes3.txt
-----BEGIN VOTE-----
Y:3:r16bGySCm5ycJFbqaH1eN4pKuEcaecXBRzA:1592926614:H75MVXvyCEPko6WT48WA0fRAVozKQbS7S2Y4tPhGJ7xBZxyCmfC03rFa/cEFcnFyvAC4Aj9CM8kBaiAdmdvwv2g=
-----END VOTE-----
-----BEGIN VOTE-----
Y:3:r17bxNVqW8o2BmXy6CEZnkDDyXcTHFWyQbB:1592932858:H/07cxON2G8vRo0oPKz9i5pJpmmqjyieaxdKm4c59u0ldoJQhNsgOv2/1VtATtINVkofAyc6UOEvch9jr32w67Q=
-----END VOTE-----
-----BEGIN VOTE-----
Y:3:r18fiaRwkHZNy2n37wRLrH4SuFTfJJCS2HV:1592983356:H0t8hSMbwLPCpHA2K+x6bkGen/BCgBkonsODj9qyiC4KP6mPAiV+Gd7o/A/piOjP4k0yUAIybmoV3Z+FlDMZ+Lo=
-----END VOTE-----
-----BEGIN VOTE-----
Y:3:r13fwb1K9sK6YYoTf22c6CJPxequKMxcA7L:1592983765:HzA1YJdWX00EUpFm19+0DasO6GRDtfBPiYMLWVoOZd6pGO+w2k8fN/oRHfb1CXCm0rwpXCWeq1wJNgZyQfvXYA4=
-----END VOTE-----
```

```
cat list_of_votes3.txt | grep -v "BEGIN VOTE" | grep -v "END VOTE" | ./verify.sh 3
jq installed... proceeding
curl installed... proceeding

--------------------------------------------------------
| Final Tally for GitHub Issue: 3
--------------------------------------------------------
| YES: 4
|-------------------------------------------------------
| NO: 0
--------------------------------------------------------
```

This grant is approved. Once the terms of the proposal are met, we will release the RES reward from the dev fund.

**Comment by ghost** [comment]
Luke , my opinion , this grant is too big , it is a full job and it comes too early. We would leave it open unless another person takes it

**Comment by MEtoolel** [comment]
My opinion , this grant is  not  big  Marketing is more about profit

**Comment by ghost** [comment]
I agree and that is why we will not take it at the moment.  It is a huge task for 2 people and the only goal is to increase resdex users. With the current resdex and without having a proper wallet and so on it is a huge task. I proposed a different idea
